scripts/globals/spells/trust/excenmille.lua

```diff
@@ -2,12 +2,20 @@
 -- Trust: Excenmille
 -----------------------------------------
 require("scripts/globals/trust")
+require("scripts/globals/zone")
 -----------------------------------------
 
 function onMagicCastingCheck(caster, target, spell)
     return tpz.trust.canCast(caster, spell, 1004)
 end
 
 function onSpellCast(caster, target, spell)
+    local SandoriaFirstTrust = caster:getCharVar("SandoriaFirstTrust")
+    local zone = caster:getZoneID()
+
+    if SandoriaFirstTrust == 1 and (zone == tpz.zone.WEST_RONFAURE or zone == tpz.zone.EAST_RONFAURE) then
```

Comment:
All of the first-trust quests name drop the two nearest starter zones, and I think the reminder text also tells you to go there. Also, all of the wikis mention it, but that's not such concrete data. I didn't test on retail to see if it HAS to be those zones, or if summoning anywhere will work.

Reply:
@ibm2431 wishlist item: Teo wants script hooks _for players_ instead of just mobs. so we can stuff global player related things into player.lua in an appropriately named function. We have no hooks for players right now, just binding for on level up/down, zonage/login, and charCreation.

This here for trusts are better off in the spell script I think, but it reminded me. figured I better say something before I forget.